juwai/ansible-role-laravel
# Create Role

## Summary of changes
- Create folders
- Generate .env config file
- Generate Nginx config file
## How to Test
1. `$ cd tests`
2. `$ vagrant up`
3. Don't see errors while provisioning (but if you do, try provisioning again)

/fyi: @wanleung @KixPanganiban: Please review.
